dcsazure_AzureSQL_to_AzureSQL_prof_pl/README.md

```diff
@@ -0,0 +1,80 @@
+# dcsazure_AzureSQL_to_AzureSQL_prof_pl
+## Delphix Compliance Services (DCS) for Azure - AzureSQL to AzureSQL Profiling Pipeline
+
+This pipeline will perform automated sensitive data discovery on your AzureSQL Instance.
+
+### Prerequisites
+1. Configure the hosted metadata database and associated Azure SQL service (version `V2024.01.01.0`+).
+1. Configure the DCS for Azure REST service.
+1. Configure the AzureSQL linked service.
```

Comment:
Nit: Numbering is incorrect.

Reply:
It is rendered correctly in the preview 
<img width="1160" alt="image" src="https://github.com/user-attachments/assets/57b956b6-0622-488d-be76-bb510610cbed">

Reply:
I'm curious to know as to why you had to use `1.`, `1.` numbering method instead of `1.`, `2.`. The latter makes the most sense, so I wonder if there's any issue using that.

Reply:
:) Copy pasted existing. I can update.

Reply:
Incremented the numbers